### PR TITLE
Support Vietnamese

### PR DESCRIPTION
**Pull request by duyetpt**
Support translate number to words for Vietnamese

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## 0.11.3 (Next)
 
 ### Features
-  * Your contribution here.
```
Comment (jlduran) [inline CHANGELOG.md]: Please, do not remove this line.

File: lib/numbers_and_words/strategies/figures_converter/languages/vi.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          end
+
+          def hundreds?
+            @figures.hundreds ||
+              (@figures.size == 3 && (@figures.teens || @figures.tens || @figures.ones))
+          end
+
+          def hundreds(options = {})
+            if @figures.hundreds.to_i.zero?
+              translate :hundreds, 0, options
```
Comment (jlduran) [inline lib/numbers_and_words/strategies/figures_converter/languages/vi.rb]: Coveralls is complaining that this line is not getting tested.

File: CHANGELOG.md (hunk 1 above)
Comment (duyetpt) [reply CHANGELOG.md]: I will revert it as default.

File: lib/numbers_and_words/strategies/figures_converter/languages/vi.rb (hunk 2 above)
Comment (duyetpt) [reply lib/numbers_and_words/strategies/figures_converter/languages/vi.rb]: I cover it in commit *ff7d3fac9990536aca08207f931bdc0749f5c1ff*

**Comment by jlduran** [comment]
Thank you!

**Pull request merged**